AggregateMetrics/AggregateMetrics.Tests/AzureWebApp/AzureWebAppTest.cs

```diff
@@ -4,19 +4,23 @@
     using System.Diagnostics;
     using Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp;
     using Microsoft.VisualStudio.TestTools.UnitTesting;
-    
+    using Microsoft.ApplicationInsights.DataContracts;
     [TestClass]
     public class UnitTestAzureWeb
     {
         [TestMethod]
-        public void TestGetValueAndReset()
+        public void GetCounterValue()
```

Comment:
The name of this test method should tell me what is being tested, e.g. "TestFlexiblePerformanceCounterGaugeMetricTelemetryReportsValueGreaterThanZero" or whatever is appropriate.  It's okay/good for test methods to have long names that fully describe what is being tested, that way when viewing test results in summary form it is clear what/where failures occur.